test/services/entity_builder.js

```diff
@@ -404,13 +404,17 @@ describe('Entity Builder', () => {
       const params = {
         createdBy: anAddress,
         page: '2',
-        perPage: '10'
+        perPage: '10',
+        fromTimestamp: '15123123123',
+        toTimestamp: '15123323123'
       };
       const validatedParams = entityBuilder.validateAndCastFindAssetsParams(params);
       expect(validatedParams).to.deep.equal({
         createdBy: anAddress,
         page: 2,
-        perPage: 10
+        perPage: 10,
+        fromTimestamp: 15123123123,
+        toTimestamp: 15123323123
```

Comment:
Maybe smaller values to make fact that they are different, easier noticeable?

Reply:
Wanted to reassemble timestamps, however you may be right